scene/2d/physics/collision_shape_2d.cpp

```diff
@@ -232,7 +227,18 @@ real_t CollisionShape2D::get_one_way_collision_margin() const {
 	return one_way_collision_margin;
 }
 
+#ifdef DEBUG_ENABLED
+
```

Comment:
When spanning 50+ lines and multiple functions, I tend to prefer it with some spacing, as otherwise the start and end of the scope can be hard to spot at a glance.

Not a policy but I wouldn't go out of our ways to point out such things either in style reviews, unless it's inconsistent with surrounding code.

Reply:
That's fair, I find it easier to read myself when it's attached and it's AFAIK the majority of existing code but good points

Reply:
* Probably these `#ifdef`s were not needed at all, see #100317. But I was too late to answer here.